### ISSUE TITLE
Bug in feedback session opening reminder test

### ISSUE THREAD

**Issue by wkurniawan07** [TEAMMATES/repo#4185]
![image](https://cloud.githubusercontent.com/assets/7261051/9539546/7f37975c-4d85-11e5-883f-c70f5f9f5886.png)
There's no checking whether the session enables opening email or not.
This bug is hidden all these while because the only test that caters to it _gives a false positive_.

**Comment by kanghj** [comment]
If I'm not wrong, this was intentionally done in #2878, it should not be possible to enable opening emails anymore unless there's a bug somewhere, and perhaps this was not documented well.

**Comment by wkurniawan07** [comment]
Ah nice, some new insights. Anyway, here's how I uncover this:
1. Go to `FeedbackSessionOpeningReminderTest` and try to run it. It passes, but see how it passes: because of an unexpected (or expected...?) AssertionError.
2. The AssertionError was caused by the feedback session name not matching the expected. That's how the test passes, instead of the line 148 `session2.isOpeningEmailEnabled = false;` (removing this line does _not_ affect the test). I got a feeling something was seriously wrong the moment this happens.
3. A little more exploration and I got what I posted up there.
